ledger/acctupdates.go

```diff
@@ -1673,6 +1448,15 @@ func (au *accountUpdates) postCommit(ctx context.Context, dcc *deferredCommitCon
 		}
 	}
 
+	// clear the backing array to let GC collect data
+	const deltasClearThreshold = 1000
```

Comment:
Why 1000? Should this be initialized by the consensus setting?

Reply:
no, it does not have anything with consensus, it is just a memory optimization:
1. When deltas grow and have say couple thousand rounds, the backing array might be suitable for 1k+ elements more.
2. When we committed, we sliced but the backing array holds pointers to delta elements.
3. Without clearing it we might have such a large chunk unless the next backing array reallocation. 1000 is kind of a number where we might see effects of holding datas too long.

Reply:
can we declare this as a global constant / derive this value from other constants / add a comment explaining this value?

Reply:
Will make a comment — at 1000 rounds * 4 seconds that's >1 hour, the time to generate a catchpoint likely won't ever exceed this constant